### PR TITLE
feat(ui): adding footer

### PR DESCRIPTION
**Pull request by MehdiHan**
Adding footer markdown directly in Layout component

### CODE REVIEW

**Comment by MehdiHan** [comment]
Adding footer markdown directly in Layout Component

**Comment by tony-go** [comment]
Thanks @MehdiHan 👏 

I'll review it asap ^^

File: components/Layout.tsx [hunk 1]
```diff
@@ -6,8 +6,21 @@ interface Props {
 
 export default function Layout({ children }: Props) {
   return (
-    <div className="flex min-h-screen md:p-20 bg-gray-800">
+    <div className="flex flex-col items-center min-h-screen md:p-20 bg-gray-800">
       <div className="container m-auto">{children}</div>
+      <footer>
+        <p className="text-gray-200 font-thin font-sans">
+          🕸 Powered by NodeSecure -{" "}
```
Comment (tony-go) [inline components/Layout.tsx:13]: I was thinking about the logo of the organization.

**Review by tony-go — CHANGES_REQUESTED** [review]
Thanks a lot for this 🙌 👏 

Didn't check the render yet. Could you join a screenshot?

File: components/Layout.tsx [hunk 2]
```diff
@@ -6,8 +6,21 @@ interface Props {
 
 export default function Layout({ children }: Props) {
   return (
-    <div className="flex min-h-screen md:p-20 bg-gray-800">
+    <div className="flex flex-col items-center min-h-screen md:p-20 bg-gray-800">
       <div className="container m-auto">{children}</div>
+      <footer>
```
[the remark this answers is not in the document]
Comment (MehdiHan) [reply components/Layout.tsx:11]: Np i'll fix it today

**Comment by MehdiHan** [comment]
Hi @tony-go, 

Changes requested are done, i'm available if you need something else about this issue.
Feel free to ask me.

screenshoot_01
screenshoot_02

**Review by fraxken — APPROVED** [review]
LGTM

**Comment by Kawacrepe** [comment]
lgtm ! Thank's for your contribution

**Review by tony-go — APPROVED** [review]

**Pull request merged**